client/fetch-account-history.py

```diff
@@ -29,7 +29,7 @@ def decode_cid(cid_raw):
     if digest_raw.startswith('0x'):
         digest = b58encode(bytearray.fromhex(digest_raw[2:])).decode("utf-8")
     else:
-        digest = digest_raw
+        digest = b58encode(digest_raw).decode("utf-8")
```

Comment:
now it is consistent, because it encodes to base58 both times, like our rust code does